controllers/drplacementcontrol.go

```diff
@@ -1513,6 +1505,19 @@ func (d *DRPCInstance) generateVRG(repState rmn.ReplicationState) rmn.VolumeRepl
 	return vrg
 }
 
+func (d *DRPCInstance) availableS3Profiles() []string {
```

Comment:
(nit) style
- longer var names, though in this case function is short and hence acceptable
- would change the if block (at 1513) to continue and append outside the if block, though it does not change nest levels, handles no-ops earlier than actual operations

Reply:
np, will update in next version.